sdk/program/src/stake/instruction.rs

```diff
@@ -228,8 +228,11 @@ pub enum StakeInstruction {
     /// # Account references
     ///   None
     ///
-    /// The minimum delegation will be returned via the transaction context's returndata.
-    /// Use `get_return_data()` to retrieve the result.
+    /// The minimum delegation will be returned via the transaction context's returndata.  Programs
+    /// should use the [`get_minimum_delegation()`] helper function instead of calling this
+    /// function directly.
```

Comment:
```suggestion
    /// Returns the minimum delegation as a little-endian encoded u64 value.
    /// Programs can use the [`get_minimum_delegation()`] helper function to invoke and
    /// retrieve the return value for this instruction.
```

Reply:
The instruction doesn't return a u64 though. Do you think that'll be an issue for developers?

Reply:
I think that it's understood now but even in the future developers shouldn't need to think about how the return value is actually acquired for an invocation in my opinion